interfaces/automated-testing/scd/scd.yaml

```diff
@@ -36,6 +36,25 @@ components:
           enum: [Starting, Ready]
           example: Ready
 
+    InjectFlightResponse:
+      type: object
+      required:
+        - results
+      properties: 
+        result: 
+          description: >-
+            The result of the flight submission
+
+              - `Planned`: The flight submission data was correct and the operational intent was successfully accepted by the DSS.
+
+              - `Rejected`: The flight submission data was incorrect and / or the USS was not able to successfully submit the operational intent to the DSS.
+
+              - `Conflict`: The flight submission data was correct but an operational intent could not be created because of conflicting operational intent.
```

Comment:
Recommend: `The flight submission data was valid, but the flight could not be authorized because of a disallowed conflict with another flight.`  And, since it is also possible to have a conflict with a constraint, let's make the enum value a bit more specific; e.g., `ConflictWithFlight`.

Reply:
I have now added the ConflictWithFlight response